### ISSUE TITLE
Specify either an HTTP or a DNS validation method.

### ISSUE THREAD

**Issue by Adambean** [aloopkin/WinCertes#12]
**Describe the bug**
When trying to ask for a wildcard certificate the tool returns an error:

`Specify either an HTTP or a DNS validation method.`

It then shows the usage with all the arguments and options, but none of them specify whether HTTP or DNS should be used.

**To Reproduce**
`wincertes -d "*.example.com"`

**Desktop (please complete the following information):**
 - OS: Windows Server
 - Version 2008 R2

**Comment by aloopkin** [comment]
If you want to issue a wildcard certificate from let's encrypt, you need to use DNS validation. Two plugins are available for the moment, see https://github.com/aloopkin/WinCertes/blob/master/Registry.md

**Comment by Adambean** [comment]
I did set the registry keys there already. (though I'm not fussed on a DNS writable password being stored there)

Did I miss an additional plugin download?

**Comment by aloopkin** [comment]
Don't forget to set DNSValidatorType (acme-dns or win-dns)

**Comment by Adambean** [comment]
How did I not see `DNSValidatorType` 🙄 Thanks!

Getting closer, though I accept this is a separate issue. At first I was getting a crash with this:

```
Failed to register and validate order with CA: User credentials cannot be used for local connections

Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at WinCertes.Program.Main(String[] args)
```

I tried deleting the 2 authentication keys to see if it would just use the running user, because a password written in the registry is a terrible place to keep it. Looks like it got further, but still crashed:

```
Impossible to bind on port 80. A program is probably already listening on it.
Generated orders and validated challenges for domains: *.example.com

Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at WinCertes.Program.Main(String[] args)
```

**Comment by aloopkin** [comment]
Can you please use the WinCertes Debug version (separate download), because i don't have enough logs there to help you.

**Comment by Adambean** [comment]
Sure, this is probably what you're after:

```
Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at WinCertes.Program.Main(String[] args) in C:\Users\Alcide Loopkin\Documents\Visual Studio 2017\Projects\WinCertes\WinCertes\Program.cs:line 263
```

**Comment by aloopkin** [comment]
Yes thanks!
Fixed and released as 1.1.3

**Comment by Adambean** [comment]
Excellent, thank you. I can confirmed this resolved the issue for me. 3 certificates, one of which is wildcard, all issued and installed successfully.

Thanks again!
